Bindings/bindings.json

```diff
@@ -455,6 +455,20 @@
                     ]
                 },
                 {
+                    "name": "subscriptionName",
+                    "value": "string",
+                    "defaultValue": "mysubscription",
+                    "required": true,
+                    "label": "Subscription name",
+                    "help": "This is the name of the subscription from which the message will be read.",
+                    "validators": [
+                        {
+                            "expression": "^[0-9a-z][a-z0-9_.-]{1,48}[0-9a-z]$",
```

Comment:
Got it from the old portal, It should allow uppercase characters though will update it.